### ISSUE TITLE
Unable to save Billing Address

### ISSUE THREAD

**Issue by vladimir-8** [coopcycle/coopcycle-web#4645]
**Additional context**

https://github.com/user-attachments/assets/56df4961-f21e-4bda-b924-9716d40f976c

**Comment by Paul-Eraman-CoopCycle** [comment]
Can you check if this is still broken?

**Comment by vladimir-8** [comment]
Yes, it's still happening. The address is actually saved, it's only not displayed until the toggle is enabled:
![Image](https://github.com/user-attachments/assets/38eb5fbe-8e28-4cf2-9da0-56722266933e)

![Image](https://github.com/user-attachments/assets/958aa217-a80e-4af7-a9a5-5395cf1dcec7)

**Comment by Paul-Eraman-CoopCycle** [comment]
wait, but isnt that supposed to be the case? like, option 1 is that your delivery and your billing addresses are the same, and if not, then toggle button

**Comment by vladimir-8** [comment]
Yes, but check the video what happens: https://github.com/coopcycle/coopcycle-web/issues/4645#issue-2537375080

- I enable `Use different address`
- Enter address
- Click Save
- And then the toggle switches back to unselected (while the address is actually saved and visible if I toggle it again)

**Comment by Paul-Eraman-CoopCycle** [comment]
ooooooh got it
